scripts/chart-test.sh

```diff
@@ -21,7 +21,9 @@ git config "remote.$REMOTE.fetch" "+refs/heads/*:refs/remotes/$REMOTE/*"
 git fetch "$REMOTE" --unshallow &> /dev/null
 git remote update upstream --prune &> /dev/null
 
-ct lint --remote "$REMOTE" --config ct.yaml
+if ! ct lint --remote "$REMOTE" --config ct.yaml; then
```

Comment:
It's ok, but shorter would be:
`ct lint --remote "$REMOTE" --config ct.yaml || exit 1`